.pre-commit-config_template.yaml

```diff
@@ -43,6 +43,20 @@ repos:
   - id: autopep8
 - repo: local
   hooks:
+
+  - id: pylint-in-docker
+    name: pylint-in-docker
+    description: Run pylint on the code in content packs
+    docker_image:nightly: from-yml
+    docker_image: native:ga,from-yml,native:dev,native:maintenance
+    entry: pylint
+    files: Packs\/.*\.py$
+    exclude: (_test\.py)|(.vulture_whitelist\.py)$
+    args: ['--ignore=demistomock.py,CommonServerPython.py,CommonServerUserPython.py,conftest.py,.venv', '-E', '--disable=bad-option-value,unsubscriptable-object', '-d duplicate-string-formatting-argument', "--msg-template='{path}:{line}:{column}: {msg_id} {obj}: {msg}'", '--generated-members=requests.packages.urllib3,requests.codes.ok']
+    config_file_arg:
+      arg_name: '--rcfile'
+      file_name: '.pylintrc'
```

Comment:
Why do we need to have this data if we are deleting the file?

Reply:
this is for the ones in the pack directories